discovery-provider/src/tasks/index.py

```diff
@@ -620,6 +720,7 @@ def update_task(self):
             # Perform indexing operations
             index_blocks(self, db, index_blocks_list)
             logger.info(f"index.py | update_task | {self.request.id} | Processing complete within session")
+            get_track_plays(self, db)
```

Comment:
still have to review this further - is there any reason why this has to be in this core indexing flow? i believe @raymondjacobson and i chatted about a separate task or just run this in the existing index_cache.py task. 

an identity unavailability randomly bringing down or slowing indexing is not great + this flow will have to be redone entirely with true listens so it would be cleaner to not pollute this flow with centralized operations

Reply:
yeah @jowlee per the design document, this should be a separate indexing operation entirely as a separated celery task

Reply:
The indexing of plays from identity service will eventually be from a chain, but probably not the POA chain, so it's going to need an entirely separated indexing flow